modules/ROOT/pages/database-administration/standard-databases/alter-databases.adoc

```diff
@@ -19,7 +22,7 @@ ALTER DATABASE name [IF EXISTS]
 {
 SET ACCESS {READ ONLY \| READ WRITE} \|
 SET TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}] \|
-SET OPTION option value
+SET OPTION option value \|
```

Comment:
same as in the syntax page, we shouldn't have the hanging `\|` at the end of the line, and should not need to tab the examples either as there is no syntax difference between Cypher 5 and Cypher 25, both add the set default language clause (with the same syntax in difference to create that have different syntax in the two versions)